glide.lock

```diff
@@ -962,7 +962,7 @@ imports:
   - pkg/controller/servingcert
   - pkg/controller/servingcert/cryptoextensions
 - name: github.com/openshift/source-to-image
-  version: 27f0729d81fe0641486257252189b8dca856ea28
+  version: cfe7e9f503d38ceca35cc7f04fef5fdd822e8c5a
```

Comment:
This was really the only thing that moved?  You didn't cherry-pick your folder, right?

Reply:
no - I manually backed out several updates that were unrelated to s2i. This PR is to merge in a specific security-related fix.

Reply:
> no - I manually backed out several updates that were unrelated to s2i. This PR is to merge in a specific security-related fix.

That's a timebomb for everyone following you and we have to hope you got the cherrypicking right.  Please include everything.

Reply:
I'll close and make a new PR as bump(*) then.